toolkit/tools/internal/exe/exe.go

```diff
@@ -40,6 +40,11 @@ func OutputDirFlag(k *kingpin.Application, doc string) *string {
 	return k.Flag("output-dir", doc).Required().String()
 }
 
+// LogColorFlag registers a log color flag for k and returns the passed value
+func LogColorFlag(k *kingpin.Application) *string {
+	return k.Flag(logger.ColorFlag, logger.ColorFlagHelp).String()
```

Comment:
Let's also enum the 3 possible values. See how `LogLevelFlag` does it for instance.